BUILDING.md

```diff
@@ -65,22 +72,29 @@ The build output will be in the directory you specified with `do config`.
 
 ### Install Android SDK
 * Unzip the [Android SDK](https://dl.google.com/android/repository/tools_r25.2.3-macosx.zip) to a directory of your choosing.
+  * Instead, if you have Homebrew, you can install via `brew install android-sdk` and the default installation location would be `$HOMEBREW_PREFIX/opt/android-sdk`.
 * Using `tools/android` download:
   * Android 5.0.1 (API 21)
     * SDK Platform
   * Android SDK Build-tools 21.1.2
 
 ### Install Android NDK
 * Unzip the [Android NDK](https://dl.google.com/android/repository/android-ndk-r13b-darwin-x86_64.zip) to a directory of your choosing.
+  * Instead, if you have Homebrew, you can install via `brew install android-ndk` and the default installation location would be `$HOMEBREW_PREFIX/opt/android-ndk`.
 
-### Install [CMake](https://cmake.org/files/v3.7/cmake-3.7.1-Darwin-x86_64.dmg).
+### Install [CMake](https://cmake.org/files/v3.7/cmake-3.7.1-Darwin-x86_64.dmg)
+  * Instead, if you have Homebrew, you can install via `brew install cmake`.
```

Comment:
The version of the dependencies use also worries me a little. I've seen different behavior between moderately recent versions of CMake, hence the explicit links.